langserver/hover.go

```diff
@@ -47,6 +48,26 @@ func initializeFunctionDocumentation() http.FileSystem {
 	return ret
 }
 
+func compatibleVersion(originalVersion, requiredVersion string) bool {
+	if originalVersion == "" {
+		return false
+	}
+
+	originalVersionArray := strings.Split(originalVersion, ".")
+	requiredVersionArray := strings.Split(requiredVersion, ".")
+	minLength := int(math.Min(float64(len(originalVersionArray)), float64(len(requiredVersionArray))))
+
+	for i := 0; i < minLength; i++ {
+		if originalVersionArray[i] > requiredVersionArray[i] {
```

Comment:
You're comparing strings here. That doesn't do what you expect.

https://play.golang.org/p/e5xHx2aOKqI

It might make more sense to use a existing library for version comparison, e.g. https://godoc.org/github.com/blang/semver .